client/blocks/signup-form/social.jsx

```diff
@@ -58,8 +58,20 @@ class SocialSignupForm extends Component {
 		} );
 	};
 
-	// eslint-disable-next-line no-unused-vars
-	handleGitHubResponse = ( response ) => {};
+	handleGitHubResponse = ( accessToken, triggeredByUser = true ) => {
+		if ( ! triggeredByUser && this.props.socialService !== 'github' ) {
+			return;
+		}
```

Comment:
Similarly as with the comment above: Since there doesn't seem to be any need for the `triggeredByUser` check at the moment, we could remove it and check just whether the `accessToken` is available.